tasklib/tests.py

```diff
@@ -1020,7 +1028,7 @@ def test_complex_eoy_conversion(self):
             hour=23,
             minute=59,
             second=59
-            )) - datetime.timedelta(0,4 * 30 * 86400)
+            )) - datetime.timedelta(0, 4 * 30 * 86400)
```

Comment:
This line can be de-dented one level, so that the opening and closing parentheses are at the same indentation level. Also the `second=59` line should have a trailing comma.

Reply:
I didn't really catch what you meant with

> can be de-dented one level, so the opening...

But I think I fixed it in 8f4fabf